src/components/common/Back/index.js

```diff
@@ -0,0 +1,17 @@
+import { PrimeIcons } from 'primereact/api';
+import './styles.scss';
+const Back = ({ onBack }) => {
+  return (
+    <div
+      className="clickable"
+      onClick={onBack}
+      onKeyDown={e => e.key === 'Enter' && onBack()}
```

Comment:
To improve readability, migrate this to a `handleKeyDown` function

Reply:
Agree!